packages/presentation/lib/src/app/widgets/app_navigation_bar.dart

```diff
@@ -5,10 +5,10 @@ import 'package:presentation/utils/images_container.dart';
 
 class AppNavigationBar extends StatelessWidget {
   const AppNavigationBar({
-    Key? key,
     required this.bottomNavIndex,
     required this.bloc,
-  }) : super(key: key);
+    Key? key,
```

Comment:
not really you need to change like
<img width="444" alt="Screenshot 2022-09-27 at 18 04 46" src="https://user-images.githubusercontent.com/99738758/192563839-ccecb29b-a676-4f1d-943f-925d959a9766.png">
<img width="323" alt="Screenshot 2022-09-27 at 18 05 04" src="https://user-images.githubusercontent.com/99738758/192563848-2e4c406f-bec7-466b-b64d-798b6a01f31d.png">